custom_components/tuya_local/devices/woods_milan_airconditioner.yaml

```diff
@@ -0,0 +1,69 @@
+name: Woods Milan Air Conditioner
+products:
+  - id: i5rv3ykmtfcagw1t
+    name: Wood's Milan 7K
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      persists: true
+      mapping:
+        - dps_val: false
+          value: "off"
+          icon: "mdi:hvac-off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: auto
+              icon: mdi:thermostat-auto
+              value: auto
+            - dps_val: cold
+              icon: "mdi:snowflake"
+              value: cool
+            - dps_val: wind
+              icon: "mdi:fan"
+              value: fan_only
+            - dps_val: wet
+              icon: "mdi:water"
+              value: dry
+            - dps_val: hot
+              value: heat
+              icon: "mdi:heat-wave"
+    - id: 2
+      name: temperature
+      type: integer
+      range:
+        min: 15
+        max: 31
+      persists: true
```

Comment:
persist is singular, but the default is true so it is not needed here.  It is only required when you want dps to report as None when the device stops sending them.